Program.cs

```diff
@@ -1,25 +1,41 @@
 ﻿const string filePath = "input.txt";
-
+int letter_count = 0;
+int DigitCount = 0;
 try
 {
-    if (File.Exists(filePath))
+    using var reader = new StreamReader(filePath);
+
+    while (!reader.EndOfStream)
     {
-        var content = File.ReadAllText(filePath);
 
-        int letterCount = content.Count(char.IsLetter);
-        int digitCount = content.Count(char.IsDigit);   
+        var line = reader.ReadLine()!;
 
-        Console.WriteLine($"The file contains {letterCount} letters and {digitCount} digits.");
-    }
-    else
-    {
-        Console.WriteLine($"File not found at path: {filePath}");
-    }
+        foreach (var character in line)
+        {
+            var helper = new Helper();
+
+            var isCharacter = helper.IsCharacterAndIfINeedToCount(character);
+            var isDigit = helper.IsDigitAndIfItIsInTheList(character);
+
+            IncreaseLetter_CountIfTrue(letter_count, isCharacter);
+            helper.IncreaseDigitCount(DigitCount, isDigit);
+        }
+        
+        void IncreaseLetter_CountIfTrue(int letter_countt, bool isCharacter)
+        {
+            letter_count = isCharacter ? ++letter_count : letter_count;
+        }
+    }       
 }
-catch (Exception ex)
+catch (FileNotFoundException)
 {
-    Console.WriteLine($"An error occurred: {ex.Message}");
+    Console.WriteLine($"File was not found in provided path");
 }
 
+Console.WriteLine($"The file contains {letter_count} letters and {DigitCount} digits.");
+Console.ReadKey();
+
+Console.WriteLine($"The file contains {new Helper().GetResult()}");
 Console.WriteLine("\nPress any key to exit...");
 Console.ReadKey();
+return;
```

Comment:
return siuo atveju yra nereikalingas, nes kai Main() metodas, pasieks pabaiga, programa automatiskai baigs darba

Reply:
nice, matau iš tavo code review kad tu parašai komentarą ir iškart spaudi submit comment. Geriau parašyk visus komentarus ir tada paspausk submit comment. Kadangi kodo owneriui eina mail kad tu parašei kažką ir aš tada būnu distractintas ir neduok dieve pradėsiu taisyt kol tu komentuoji tada bus double effort ir neefektyvu

Reply:
Negaliu apatinio komentaro pakomentuot, bet jei netrukdytu tie failai buildui viskas ok butu ? kas ten per failai is vis ?